recipes/eagle2/build.sh

```
@@ -0,0 +1,17 @@
+    CXXFLAG="$CXXFLAGS" \
+    LDFLAG="$LDFLAGS" \
+    PREFIX="${PREFIX}" \
+    HTSLIB_INC="$PREFIX" \
+    HTSLIB_LIB="-lhts" \
+    BOOST_INC="/usr/include"\
+    BOOST_LIB_IO="-lboost_iostreams" \
+    BOOST_LIB_PO="-lboost_program_options" \
+    BOOST_LIB_SE="-lboost_serialization" \
+    ;
+install
+popd
```

Shellcheck

[warning] Use 'popd ... || exit' or 'popd ... || return' in case popd fails.

(SC2164)